debug_src/debug.js

```diff
@@ -0,0 +1,78 @@
+const preact = require('../');
+
+const original = {
+	h: preact.h
+};
+
+const config = {
+	undefinedComponents: true,
+	stringRefs: true,
+	arrayChildren: true
+};
+
+preact.h = function(nodeName, attributes) {
```

Comment:
Could this be built on Preact's existing `options.vnode` hook rather than patching `h` directly? Then it should work for `createElement` as well. This would also be consistent with how the devtools module works - hooking into Preact's global hooks.